lesson_14/homework_14.py

```diff
@@ -12,4 +12,39 @@
     Протилежні кути ромба завжди рівні, тому при заданому значенні кут_а, 
     значення кут_б обчислюється автоматично.
     Для встановлення значень атрибутів використовуйте метод __setattr__.
-"""
\ No newline at end of file
+"""
+
+class Rhomb:
+    def __init__(self, side_a: float, angle_a: float, angle_b: float = None):
```

Comment:
```suggestion
    def __init__(self, side_a: float, angle_a: float, ):
```
Кут Б може бути розрахований автоматично